console/src/main/scripts/plugins/accessPortal/vendor/js/redhat_access_angular_ui.js

```diff
@@ -3522,7 +3522,7 @@ angular.module('RedhatAccess.cases').controller('EditGroup', [
         $scope.accountNumber = null;
         $scope.isUsersPrestine = true;
```

Comment:
This JS is not ours, it's a library from Red Hat Access, which is not bowerized.